recipes-graphics/imx-gpu-apitrace/imx-gpu-apitrace/0002-cmake-Use-PythonInterp-instead-of-Python3.patch

```diff
@@ -0,0 +1,225 @@
+From 762587079dea1c24de95c38c271ed1692c8f2a18 Mon Sep 17 00:00:00 2001
+From: Jose Fonseca <jfonseca@vmware.com>
+Date: Sun, 14 Apr 2019 22:22:24 +0900
+Subject: [PATCH 2/7] cmake: Use PythonInterp instead of Python3.
+
+As the latter is not in widely available CMake versions.
+
+Upstream-Status: Pending
```

Comment:
It seems to be a backport, isn't it?

Reply:
Correct! This one is ported, would drop it now.